sklearn/utils/tests/test_tags.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+from sklearn.base import BaseEstimator
+from sklearn.utils._tags import (
+    _DEFAULT_TAGS,
+    _safe_tags,
+)
+
+
+class EstimatorNoTags:
```

Comment:
nitpick, this name sounds French to my ears :)

```suggestion
class NoTagsEstimator:
```

Note: do not apply this suggestion without modifying the code accordingly.